pd-client/client.go

```diff
@@ -297,6 +297,13 @@ func (c *client) tsLoop() {
 			if err != nil {
 				log.Errorf("[pd] create tso stream error: %v", err)
 				cancel()
+
+				n := len(c.tsoRequests)
+				for i := 0; i < n; i++ {
+					req := <-c.tsoRequests
+					req.done <- errors.Trace(err)
+				}
+
```

Comment:
Is it possible that new requests send to `tsoRequests` channel after responded all existing requests?